src/test/java/org/opensearch/sdk/TestSDKClient.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk;
+
+import org.junit.jupiter.api.Test;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.opensearch.indices.Alias;
+import org.opensearch.client.opensearch.indices.CreateIndexRequest;
+import org.opensearch.client.transport.OpenSearchTransport;
+import org.opensearch.client.transport.TransportException;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class TestSDKClient {
+    private OpenSearchTransport transport = new FailingTransport();
+
+    @Test
+    public void testCreateIndex() throws Exception {
+        OpenSearchClient client = new OpenSearchClient(transport);
+
+        //tag::builders
+        assertThrows(TransportException.class, () -> client.indices().create(
```

Comment:
I see a test here for failing, but where is the test that it works?

Reply:
We don't have a running OS to test when it works.

Reply:
Can we open an issue to add integration tests? 
We should be able to spin up opensearch nodes and make calls. 

@dbwiddis question makes a lot of sense, a.k.a How do we really know this client works other than manual testing?

Reply:
https://github.com/opensearch-project/opensearch-sdk-java/issues/97